pom.xml

```diff
@@ -71,7 +71,8 @@ under the License.
     </distributionManagement>
 
     <properties>
-        <flink.version>1.14.4</flink.version>
+        <flink.version>1.15.2</flink.version>
+        <flink.submodule.postfix></flink.submodule.postfix>
```

Comment:
The property is empty?

Reply:
Yes, the flink 1.15.2 need not scala suffix. I will try to remove this config and 1.14 profile.